docker-compose.yml

```diff
@@ -16,6 +16,17 @@ services:
     command: sh -c "cd /faf-stack && sh scripts/init-volumes.sh"
 
   #
+  # Coturn server for proxying between players
+  # It uses net: "host" for performance reasons.
+  #
+  coturn:
+    image: bprodoehl/turnserver
+    network_mode: "host"
+    environment:
+    - SKIP_AUTO_IP=true
+    - EXTERNAL_IP=${EXTERNAL_IP}
```

Comment:
* The service should be called `faf-coturn` to keep up with the naming convention
* The environment variables should be defined in `config.template/faf-coturn/faf-coturn.env`, where `EXTERNAL_IP` should be set to something generic like `127.0.0.1`
* `EXTERNAL_IP` can then be removed from `.env`